src/Files.App/Views/SplashScreenPage.xaml

```diff
@@ -42,6 +42,7 @@
 			x:Name="SplashScreenLoadingProgressRing"
 			Margin="0,0,0,48"
 			VerticalAlignment="Bottom"
+			FlowDirection="LeftToRight"
```

Comment:
What happens if this isn't hardcoded, does the progress ring spin in the wrong direction?

Reply:
inverse spin